src/storage/txn/store.rs

```diff
@@ -647,7 +659,7 @@ mod tests {
     fn test_isolation_multi_inc() {
         const THREAD_NUM: usize = 4;
         const KEY_NUM: usize = 4;
-        const INC_PER_THREAD: usize = 100;
+        const INC_PER_THREAD: usize = 10;
```

Comment:
@ngaut It costs too much time due to the naive implementation of `EngineBtree::snapshot()`. It may be rolled back after I finding a better in-memory Engine.